params/protocol_params.go

```diff
@@ -69,6 +69,32 @@ const (
 	MemoryGas        uint64 = 3     // Times the address of the (highest referenced byte in memory + 1). NOTE: referencing happens on read, write and in instructions such as RETURN and CALL.
 	TxDataNonZeroGas uint64 = 68    // Per byte of data attached to a transaction that is not equal to zero. NOTE: Not payable on data of calls between transactions.
 
+	// These have been changed during the course of the chain
+	CallGasFrontier        uint64 = 40  // Once per CALL operation & message call transaction.
+	CallGasEip150          uint64 = 700 // Static portion of gas for CALL-derivates after EIP 150 (T.W)
+	BalanceGasFrontier     uint64 = 20  // The cost of a BALANCE operation
+	BalanceGasEip150       uint64 = 400 // The cost of a BALANCE operation after Tangerine Whistle
+	ExtcodeSizeGasFrontier uint64 = 20  // Cost of EXTCODESIZE before EIP 150 (T.W)
+	ExtcodeSizeGasEip150   uint64 = 700 // Cost of EXTCODESIZE after EIP 150 (T.W)
+	SloadGasFrontier       uint64 = 50
+	SloadGasEip150         uint64 = 200
+	ExtcodeHashGas         uint64 = 400  // Cost of EXTCODEHASH (introduced in Constantinople)
+	SuicideGasEip150       uint64 = 5000 // Cost of SELFDESTRUCT post T.W
+
+	// EXP has a dynamic portion depending on the size of the exponent
+	ExpByteFrontier uint64 = 10 // was set to 10 in Frontier
+	ExpByteEip158   uint64 = 50 // was raised to 50 during Eip158 (Spurious Dragon)
+
+	// Extcodecopy has a dynamic AND a static cost. This represents only the
+	// static portion of the gas. It was changed during EIP 150 (Tangerine Whistle)
+	ExtcodeCopyBaseFrontier uint64 = 20
+	ExtcodeCopyBaseEip150   uint64 = 700
```

Comment:
We're a bit lax on the `EIP` vs `eip` vs `Eip` names. If we want to be consistent, we should use `EIP` when it's in the middle of a name, or `eip` if it's a (private) prefix. Could we update all these names to be consistent? Imho it will help keep code clean in the long term.